CONTRIBUTING.md

```diff
@@ -8,36 +8,25 @@ In domains like automotive, robotics or gaming, a huge amount of data must be
 transferred between different parts of the system. If these parts are actually
 different processes on a POSIX based operating system like Linux, this huge
 amount of data has to be transferred via an inter-process-communication (IPC)
-mechanism.
-
-* https://projects.eclipse.org/projects/technology.iceoryx
+mechanism. Find more infos on the [Eclipse site](https://projects.eclipse.org/projects/technology.iceoryx).
 
 ## Developer resources
 
 Information regarding source code management, builds, coding standards, and
-more.
-
-* https://projects.eclipse.org/projects/technology.iceoryx/developer
-
-The project maintains the following source code repositories
-
-* https://github.com/eclipse/iceoryx
+more can be found on the [Eclipse site](https://projects.eclipse.org/projects/technology.iceoryx/developer).
```

Comment:
There is not much to find, only same backlinks?

Reply:
Removed this section